src/devices/KeyMatrix/samples/KeyMatrix.Sample.cs

```diff
@@ -0,0 +1,98 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Device.Gpio;
+using System.Linq;
+using System.Threading;
+
+namespace Iot.Device.KeyMatrix.Samples
+{
+    /// <summary>
+    /// This sample is for Raspberry Pi Model 3B+
+    /// </summary>
+    internal class Program
+    {
+        public static void Main(string[] args)
+        {
+            // get arguments
+            Console.Write("input output pins(eg. 26,19,13,6) ");
```

Comment:
not sure I understand what this message says. Perhaps "Enter output pins (default: 26, 19, 13, 6)", same comment below for inputs